mac

```diff
@@ -124,48 +123,27 @@ fancy_echo "Update heroku binary..."
 brew unlink heroku
 brew link --force heroku
 
-fancy_echo "Configuring asdf version manager..."
-append_to_shellrc "source /usr/local/opt/asdf/asdf.sh" 1
-# For the current session as well
-. /usr/local/opt/asdf/asdf.sh
+fancy_echo "Installing rbenv"
+brew install rbenv
+rbenv init
+echo 'eval "$(rbenv init -)"' >>~/.bash_profile
 
-install_asdf_plugin() {
-  local name="$1"
-  local url="$2"
 
-  if ! asdf plugin-list | grep -Fq "$name"; then
-    asdf plugin-add "$name" "$url"
-  fi
-}
+fancy_echo "Installing latest Ruby..."
+rbenv install "2.4.1"
 
-# shellcheck disable=SC1090
+fancy_echo "Installing nvm..."
+brew install nvm
 
-fancy_echo "Installing asdf plugins for ruby and node"
-install_asdf_plugin "ruby" "https://github.com/asdf-vm/asdf-ruby.git"
-install_asdf_plugin "nodejs" "https://github.com/asdf-vm/asdf-nodejs.git"
-# Import the Node.js release team's OpenPGP keys to main keyring
-~/.asdf/plugins/nodejs/bin/import-release-team-keyring
+mkdir ~/.nvm
+echo 'export NVM_DIR="$HOME/.nvm"' >>~/.bash_profile
+echo '[ -s "/usr/local/opt/nvm/nvm.sh" ] && . "/usr/local/opt/nvm/nvm.sh"' >>~/.bash_profile
+echo '[ -s "/usr/local/opt/nvm/etc/bash_completion" ] && . "/usr/local/opt/nvm/etc/bash_completion"' >>~/.bash_profile
```

Comment:
Could wrap these in a check to avoid writing to `/.bash_profile` multiple times by accident:

```sh
if [ "$NVM_DIR" != "" ]; then
	echo 'export NVM_DIR="$HOME/.nvm"' >>~/.bash_profile
	echo '[ -s "/usr/local/opt/nvm/nvm.sh" ] && . "/usr/local/opt/nvm/nvm.sh"' >>~/.bash_profile
	echo '[ -s "/usr/local/opt/nvm/etc/bash_completion" ] && . "/usr/local/opt/nvm/etc/bash_completion"' >>~/.bash_profile
fi
```

Reply:
Original `[ "$NVM_DIR"!= "" ]` was supposed to be `[ "$NVM_DIR" = "" ]`, my bad 😞